GenerateCLP/Testing/CLPExample1/CMakeLists.txt

```diff
@@ -26,7 +26,7 @@ endif()
 set(${PROJECT_NAME}_SOURCE ${PROJECT_NAME}.cxx)
 GENERATECLP(${PROJECT_NAME}_SOURCE ${PROJECT_NAME}.xml)
 add_executable(${PROJECT_NAME} ${${PROJECT_NAME}_SOURCE})
-target_link_libraries(${PROJECT_NAME} ITKEXPAT ${_additional_link_libraries})
+target_link_libraries(${PROJECT_NAME} ${_additional_link_libraries})
```

Comment:
You should test for the case when there are no `_additional_link_libraries`

```
if(_additional_link_libraries)
  target_link_libraries(${PROJECT_NAME} ${_additional_link_libraries})
endif()
```

In the commit message, consider mentioning something like this - Do not hesitate to improve the message if I misunderstood something.

```
This commit fixes the build error report below by removing the library ITKEXPAT
originally added in Slicer r317 [1] (ENH: Added examples) from the list of explicit
example dependencies. Instead, it add ITKEXPAT as a dependency of
ModuleDescriptionParser that will be linked into both the examples and any other
CLIs. It will prevent linking against incorrect library found on the system.

[1] http://viewvc.slicer.org/viewvc.cgi/Slicer4?view=revision&revision=317
```